includes/template-tags.php

```diff
@@ -552,7 +552,14 @@ function dokan_store_category_menu( $seller_id, $title = '' ) {
         $all_terms = [];
         foreach ( $product_ids as $product_id ) {
             $terms = get_the_terms( $product_id, 'product_cat' );
-            array_push( $all_terms, $terms[0] );
+
+            if ( count( $terms ) > 1 ) {
```

Comment:
do we need this check? we can easily loop through the terms if there is one or more. we need to just push the term in the array.